android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -214,6 +223,7 @@ public void play(final Integer key, final Callback callback) {
       @Override
       public synchronized void onCompletion(MediaPlayer mp) {
         if (!mp.isLooping()) {
+          setOnPlay(false, key);
```

Comment:
`onCompletion ` is called when `MediaPlayer` pause or stop? If not, it should be added in the `stop` and `pause`.

Reply:
Ah! I found that you handle `stop` and `pause` case in the sound.js! Is there any reason that you handle this on sound.js not native code?

Reply:
`stop` and `pause` has explicit callback function. So handling it in callback function would be faster than using event Emitter!